### ISSUE TITLE
In Nook Simple Touch and Ebookdroid 1.5.6 (1.5.2 and 1.5.5 also) : 1. "Pinch-zoom/unzoom or change orientation on same page repeatedly (3-4 times), the viewer crashes silently and returns to the Browser or Library Activity."

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#355]
```
In Nook Simple Touch and Ebookdroid 1.5.6 (1.5.2 and 1.5.5 also) :

1. "Pinch-zoom/unzoom or change orientation on same page repeatedly (3-4 
times), the viewer crashes silently and returns to the Browser or Library ."
2. In "go to" window - remove bookmarks buttons not work.
3. When you move to another page when a high magnification, a new page remains 
white and only after little Pinch back to shows the text.

What is Ebookdroid optimized version good for NST?
```

Original issue reported on code.google.com by `gold...@gmail.com` on 26 Oct 2012 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
First of all: Please create separate issue for each problem. So, we will treat 
this issue only for problem 1 - crashes. Please create another issues for other 
bugs.

When EBookdroid crashes it writes dump to /sdcard/.org.ebookdroid/logs/ 
Please attach those logs.
```

Original comment by `Andrei.K...@gmail.com` on 26 Oct 2012 at 7:02
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
sorry, I thought it was the subject for the Nook.I will create.
```

Original comment by `gold...@gmail.com` on 26 Oct 2012 at 7:26

Attachments:
- [logs.zip](https://storage.googleapis.com/google-code-attachments/ebookdroid/issue-355/comment-2/logs.zip)

**Comment by GoogleCodeExporter** [comment]
```
Since 1.6.x NST not supported
```

Original comment by `Andrei.K...@gmail.com` on 15 Apr 2013 at 12:52
- Changed state: **WontFix**
